src/srtctl/core/backend.py

```
@@ -0,0 +1,212 @@
+        if prof == "nsys":
+            lines.append(
+                "nsys profile -t cuda,nvtx --cuda-graph-trace=node -c cudaProfilerApi --capture-range-end stop --force-overwrite true python3 -m sglang.launch_server \\"
+            )
+        elif use_sglang:
+            lines.append("python3 -m sglang.launch_server \\")
+        else:
+            lines.append("python3 -m dynamo.sglang \\")
+
+        if config_path:
+            with open(config_path) as f:
+                sglang_config = yaml.load(f, Loader=yaml.FullLoader)
```

Ruff (0.14.8)

Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)